apmpackage/apm/changelog.yml

```diff
@@ -1,11 +1,11 @@
 # newer versions go on top
 #
 # change type can be one of: enhancement, bugfix, breaking-change
-- version: "generated"
+- version: "8.5.0"
```

Comment:
Not sure where I commented, commenting again here: this one should be left as "generated" - it will be replaced by a script. Just update the placeholder entry below.

Reply:
Updated, thanks.